service/src/main/resources/templates/ehr_narrative_statement_template.mustache

```diff
@@ -0,0 +1,13 @@
+{{#isNested}}
+<component typeCode="COMP" contextConductionInd="true">
+{{/isNested}}
+{{^isNested}}
+<component typeCode="COMP">
+{{/isNested}}
+    <NarrativeStatement classCode="OBS" moodCode="EVN">
```

Comment:
This way of doing it is correct, but there is a more compact way to write it.

`<component typeCode="COMP"{{#isNested}} contextConductionInd="true"{{/isNested}}>`